tutorial/README.md

```diff
@@ -69,12 +69,17 @@ Therefore for larger images, please use `bioformats2raw + raw2ometiff`.
 > by `raw2ometiff` - `LZW`, `zlib`, and `Uncompressed` as well as `jpeg` compression for 8 bit data. Support
 > for JPEG-2000 for >8 bit data is planned. Please open an issue if you would like this more immediately.
 
+### Indexed TIFF
```

Comment:
I'd use "Indexed OME-TIFF". I know we had a discussion about how this approach is generic but this section, and tutorial, deals specifically with our proposed Indexed OME-TIFF that we evaluated in the paper. Folks can read the `gerenate-tiff-offsets` repo if there are interested in learning more.